pyvo/astropy_samp/hub_script.py

```diff
@@ -0,0 +1,144 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+
+import copy
+import time
+import sys
+import argparse
+
+from astropy import log
```

Comment:
If you want to replace this with your own logger, please give some advice on how to do it properly.